plugin-server/src/main/ingestion-queues/batch-processing/each-batch.ts

```diff
@@ -34,7 +39,16 @@ export async function eachBatch(
             }
 
             const lastBatchMessage = messageBatch[messageBatch.length - 1]
-            await Promise.all(messageBatch.map((message: KafkaMessage) => eachMessage(message, queue)))
+            await Promise.all(
+                messageBatch.map((message: KafkaMessage) => {
+                    const eachMessageSpan = batchSpan.startChild({ op: 'eachMessage' })
```

Comment:
nit: this one might not be worth the added volume if we are billed per span. If we are billed per "transaction", keep it.